metricbeat/module/prometheus/test_prometheus.py

```diff
@@ -35,3 +36,34 @@ def test_stats(self):
         self.assertCountEqual(self.de_dot(PROMETHEUS_FIELDS), evt.keys(), evt)
 
         self.assert_fields_are_documented(evt)
+
+
+class TestRemoteWrite(metricbeat.BaseTest):
+
+    COMPOSE_SERVICES = ['prometheus-host-network']
+
+    @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    def test_remote_write(self):
+        """
+        prometheus remote_write test
+        """
+        self.render_config_template(modules=[{
+            "name": "prometheus",
+            "metricsets": ["remote_write"],
+            "period": "5s",
+            "host": "localhost",
+            "port": "9201"
```

Comment:
I wonder if we should use the `hosts` option to configure the listener as the `http.server` metricset does. Though the `statsd` module does it this way.

Reply:
I think it is hardcoded there too -> https://github.com/elastic/beats/blob/master/metricbeat/module/http/test_http.py#L53
🤔

Reply:
Oh, you are right, I was looking at the test for the json metricset. :+1: